.storybook/main.ts

```diff
@@ -18,10 +18,10 @@ const config: StorybookConfig = {
   staticDirs: ["./public"],
 
   addons: [
-    getAbsolutePath("@storybook/addon-links"),
-    getAbsolutePath("@storybook/addon-essentials"),
-    getAbsolutePath("@storybook/addon-interactions"),
-    getAbsolutePath("@storybook/addon-webpack5-compiler-swc"),
```

Comment:
works around a bug in Storybook that occurs when you try and run it on Node 22.12.0, detailed here: https://github.com/nodejs/node/issues/56127